src/rules/__tests__/no-try-expect.test.js

```diff
@@ -0,0 +1,66 @@
+import { RuleTester } from 'eslint';
+import rule from '../no-try-expect';
+
+const ruleTester = new RuleTester({
+  parser: require.resolve('babel-eslint'),
+});
+
+const errors = [
```

Comment:
Could I ask you to inline this like how the other rules do? 

I'm not against this, but when this rule is converted to TypeScript, I'll have explicitly type the array or inline the object so that TS can implicitly infer the type, so doing it now will save me a bit of extra work later :)